libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/RouteRefreshController.kt

```diff
@@ -37,8 +38,14 @@ internal class RouteRefreshController(
 
     private var requestId: Long? = null
 
+    var routeRefreshInterval = routeRefreshOptions.intervalMillis
```

Comment:
We should be able to remove this field since the value is immutable now and coming from the options.

Reply:
Yeah, also removed it from an `init`. moved it closer to the MapboxTimer initialization